src/style.css

```diff
@@ -1,3 +1,97 @@
-body {
+:root {
+  --background-color: #fff;
+  --main-color: #00acdc;
+  --border-height: 3px;
+  --font-size: 12px;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  /* stylelint-disable-next-line font-family-name-quotes */
+  font-family: 'Roboto', sans-serif;
+}
+
+.menu {
+  width: 100%;
+  background-color: var(--background-color);
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.logo {
+  margin: 20px;
+  max-height: 50px;
+}
+
+.menu_box {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+}
+
+.menu_link {
+  line-height: auto;
+  font-weight: 500;
+  text-decoration: none;
+  list-style: none;
+  gap: 20px;
+  margin: 20px;
```

Comment:
The `gap` property is not applicable to inline elements like anchor tags, which `.menu_link` seems to be. It's used with grid and flex containers. Also, `margin: 20px;` will apply to all sides. If the intention is to only have horizontal spacing, use `margin-left` and `margin-right` instead.